### PR TITLE
Badge & Button

### LINKED ISSUE #5: Badge
- [x] Badge on avatar
- [x] Badge sizes
- [x] Badge on icons

### LINKED ISSUE #6: Buttons
- [x] Basic Buttons
- [x] Outline Buttons
- [x] Link Buttons
- [x] Buttons with icons

### PR DESCRIPTION
**Pull request by thenorthkun**
Added the following components / sub-components -

Badge

- Badge on avatar
- Badge sizes
- Badge on icons

Buttons

- Basic button
- Outline Button
- Link Button
- Button with icon
- Floating action button

**Badge preview :** https://fezco-ui-git-badge-button-thenorthkun.vercel.app/components/badge/badge.html
**Button preview :** https://fezco-ui-git-badge-button-thenorthkun.vercel.app/components/button/button.html

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    background-color: var(--grey);
+}
+
+.idle{
+    background-color: #f97316;
+
+}
+
+.dnd{
+    background-color: red;
```
Comment (j24m) [inline components/badge/badge.css]: color variable ?

**Review by j24m — APPROVED** [review]
You have followed all good practices, used rems and px properly, code is good, did not really find anything that needs change. :)

File: components/button/button.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+                    </button>
+                    <button class="btn outline-dark btn-icon">
+                        <box-icon name='search-alt'></box-icon>Search
+                    </button>
+
+                </div>
+
+                <div class="code-snippet"></div>
+                <div class="divider"></div>
+
```
Comment (mehulsatardekar) [inline components/button/button.html:193]: A ui Suggestion
Capture

Here the contrast of the yellow button way to low , you can increase the contrast ratio.

**Review by mehulsatardekar — APPROVED** [review]
The rest code looks good and nice work on UI and loved the font selection for CL.

File: components/button/button.html (hunk 2 above)
Comment (thenorthkun) [reply components/button/button.html:193]: Noted.

**Pull request merged**